app/src/main/java/com/example/drawingactivity/CustomColorViewModel.kt

```diff
@@ -43,6 +43,7 @@ class CustomColorViewModel : ViewModel() {
     var defaultLayerHintFound = false
     var currentUserEnteredLayerName: String? = null
     var layerNumber: Int = LAYER_COUNT_START
```

Comment:
so I'm assuming this is part of what would help give you the hint value, what happens If you have layer 1, 2, 3, 4 and 5. The user deletes 3, either your hint would be 3 and layer Number would be wrong, or you would give 4 and layer number would be right.

Reply:
I calculate the hint by iterating through a loop for this, if I don't find the name in the list I create a hint at that value

for (i in 0 until layerList.size) {
                if (layerList[i] == defaultLayerName) {
                    layerNumber += 1
    } 
}

The constant is just a value to initialize the variable at for app startup, it doesn't impact any of the functionality for that